src/Theme/FooterTemplate.elm

```diff
@@ -15,10 +15,14 @@ view model =
             translate model.language
     in
     footer [ css [ footerStyle ] ]
-        [ h2
+        [ img [ src "/images/dfl-logo.svg" ] []
+        , h2
             []
             [ text (t FooterText)
```

Comment:
I think we should remove this h2. It was a placeholder so we could "see the footer"